src/main/telemetry/telemetry.h

```diff
@@ -59,5 +59,4 @@ bool telemetryDetermineEnabledState(portSharing_e portSharing);
 
 void telemetryUseConfig(telemetryConfig_t *telemetryConfig);
 
-#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM)
-
+#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_CRSF)
```

Comment:
I think we will be looking to clean all that up at some point in the future. there will be protocols that will be independent and then those that are not (half duplex protocols for instance) - the underlying implementation though could well be the same for both situations.

Independent protocols can be simply passed a serial port implementation of the buffer output, whilst the dependent ones can be passed a buffer output that relies on the rx for delivery (via serial or spi or whatever).